components/automate-ui/src/app/modules/policy/list/policy-list.component.html

```diff
@@ -13,6 +13,7 @@
                 <chef-thead>
                   <chef-tr>
                     <chef-th class="name-column">Name</chef-th>
+                    <chef-th class="id-column">ID</chef-th>
```

Comment:
We're adding a new column here, so we need to adapt the `nth` number used for selection table cells here: 
https://github.com/chef/automate/blob/9e32a4ca5c2966e7518a75ea67dc2f534473641c/components/automate-ui/e2e/admin.e2e-spec.ts#L167-L170

Try `chef-td:nth-child(3)` instead, and add an assertion for the newly added column, like

```typescript
 waitForElement('chef-table chef-tr:nth-child(1) chef-td:nth-child(2)') 
   .then((policyType) => { 
     expect(policyType.getText()).toBe('chef-managed-administrator'); 
   }); 
```

Reply:
@srenatus  I tried these commands
``` 
ng e2e --no-webdriver-update
sudo make e2e
```
for resolving binaries error:
```
sudo webdriver-manager update
```
after code changes to test code locally but it gives me error as in below attachment and not able to test the changes locally.

![test error](https://user-images.githubusercontent.com/10369422/97992613-ed9d1b00-1e08-11eb-86f3-f6c59abc8c38.png)

Reply:
This question is unrelated to the PR; if still having difficulty you should post this question in the #ui-team channel.